libandroid-navigation/src/main/java/com/mapbox/services/android/navigation/v5/navigation/NavigationWalkingOptions.kt

```diff
@@ -0,0 +1,73 @@
+package com.mapbox.services.android.navigation.v5.navigation
+
+import com.mapbox.api.directions.v5.WalkingOptions
+
+/**
+ * Class for specifying options for use with the walking profile.
+ */
+class NavigationWalkingOptions internal constructor(val walkingOptions: WalkingOptions) {
```

Comment:
They are independent, one in old module, another in new one. As I remember we decided don't remove `NavigationWalkingOptions` in `libdirections-offboard` because we'll need it for future features

Reply:
If those builders and DTOs are shared, they should live in base.

Reply:
they are, ptal at [model](https://github.com/mapbox/mapbox-navigation-android/pull/2300/files#diff-94369bf215c32e7df9225179dc674c7d)